sys/psa_crypto/doc.txt

```diff
@@ -112,8 +112,13 @@
  * Persistent keys will also be written into flash memory for later access. To destroy
  * them they must be explicitly deleted with the `psa_destroy_key()` function.
  *
- * @note    So far this implementation only supports volatile storage. Persistent storage
- *          will be added in the future.
+ * @note    Persistent key storage can be optionally enabled on `native` and on the `nRF52840dk`.
+ *          For this, add `USEMODULE += psa_persistent_storage` to your application makefile
+ *          or `CONFIG_MODULE_PSA_PERSISTENT_STORAGE=y` to your `app.config.test` file.
+ *          Example: `tests/sys/psa_crypto_persistent_storage`
```

Comment:
I can't comment on the unchanged lines below, but the current state of the implementation does not match the documentation: Keys are only stored persistently to flash in case the in-memory key slots are all occupied. From a user perspective, I would however expect the key material to be stored to non-volatile flash right away so that it stays accessible after reboot.

Reply:
Alright, scratch that, just had another look and found the call to persist the key in `psa_finish_key_creation`.